pkgs/applications/misc/appmenu-gtk-module/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, fetchurl, cmake, glib, gtk2, gtk3 }:
+stdenv.mkDerivation rec {
+  pname = "appmenu-gtk-module";
+  version = "0.7.3.2";
+  srcs = {
+    appmenu-gtk-module = "${fetchFromGitHub {
+      owner = "rilian-la-te";
+      repo = "vala-panel-appmenu";
+      rev = version;
+      fetchSubmodules = true;
+
+      sha256 = "0fp8d38sjxpx8xpq0hxm67p7clhc3cbag581srxba6rhdcj5brkd";
+      }}/subprojects/appmenu-gtk-module";
+
+    xinitfile = fetchurl {
+      url = "https://raw.githubusercontent.com/archlinux/svntogit-community/packages/appmenu-gtk-module/trunk/80-appmenu-gtk-module";
+      sha256 = "1g4zjqnr89achb8k97j26f0dl8wy7afm4dbdmmq60mdqlx76q02c";
+    };
```

Comment:
Don't put this into `srcs`, but just in an outside let expression, and interpolate in the install command.